vscode-lean4/src/leanclient.ts

```diff
@@ -310,6 +330,7 @@ export class LeanClient implements Disposable {
             }
             // if we got this far then the client is happy so we are running!
             this.running = true;
+            this.showingRestartMessage = false;
```

Comment:
modifying `this.showingRestartMessage` doesn't belong here, it should be localized to  the showRestartMessage method.